src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
```

Comment:
don't overuse `*`-selector, style margins and paddings for each of the element you need